test/journeys/lib/test-helpers/basic.js

```diff
@@ -0,0 +1,28 @@
+export const elements = {
```

Comment:
Could we make this file `test-helpers/space-widget/main.js` and move `meet.js` and `messaging.js` into this folder?